corehq/apps/userreports/indicators/__init__.py

```diff
@@ -104,32 +104,34 @@ def get_values(self, item, context=None):
 
 class LedgerBalancesIndicator(ConfigurableIndicator):
```

Comment:
this is not at all related to your PR but just occurred to me while reviewing so writing it somewhere. it feels like what we should do is expose a `get_case_ledgers` expression that then can play nice with all the map/reduce stuff and would be more versatile for anyone doing anything off of ledgers. i haven't totally thought through whether that could be a replacement for this functionality (might be tricky with expanding columns), but seems like it would simplify things if we could use it.

(feel free to ignore this comment)